test/e2e/serial/tests/workload_overhead.go

```diff
@@ -274,6 +274,158 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 				}
 
 			})
+
+			It("[test_id:47582][tier2] Pod pending when resources requested + pod overhead don't fit on the target node; NRT objects are not updated", func() {
```

Comment:
let's update the test_id to 53819 as we already have an It with 47582 set.

Reply:
Ack